problems/1.types+operators.js

```diff
@@ -2,19 +2,22 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  return typeof val === "number"
 }
+console.log(isNumber(4))
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
-
+function isTruthy(val) {
+  return val && true;
```

Comment:
This is an interesting approach but won't evaluate every time to a boolean. Remember we have the ! operator to get the opposite boolean so !! would have worked. Also could have done a condition like if(val) return true else false